recipes/yaml-cpp/all/conanfile.py

```diff
@@ -60,10 +68,13 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, 'lib', 'cmake'))
         tools.rmdir(os.path.join(self.package_folder, 'CMake'))
         tools.rmdir(os.path.join(self.package_folder, 'lib', 'pkgconfig'))
+        tools.rmdir(os.path.join(self.package_folder, 'share'))
 
     def package_info(self):
-        self.cpp_info.libs = tools.collect_libs(self)
-        if self.settings.os == "Linux":
+        self.cpp_info.libs = ["yaml-cpp"]
```

Comment:
Debug libraries have a `d` suffix.

Reply:
that's right. cannot understand where is 'md' from (`libyaml-cppmd`). reverted to `collect_libs` :(

Reply:
The suffix/postfix is applied here:
https://github.com/jbeder/yaml-cpp/blob/b591d8ae2ad1ff373273c3e05973adf6c46abfa8/CMakeLists.txt#L128